backend/cmd/migrate/migrations/000004_experiments_schema_changes.up.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS experiments;
+CREATE TABLE IF NOT EXISTS experiment_config (
+    id BIGINT,
+    details JSONB
+);
+CREATE TABLE IF NOT EXISTS experiment_run (
+    id BIGINT,
+    experiment_config_id BIGINT,
+    start_time TIMESTAMP WITH TIME ZONE,
+    end_time TIMESTAMP WITH TIME ZONE,
+    scheduled_end_time TIMESTAMP WITH TIME ZONE,
```

Comment:
thoughts on incorporating the range type?

Reply:
I tried to use it but it doesn't support 'TIMESTAMP WITH TIME ZONE' apparently

Reply:
it supports TIMESTAMP

Reply:
https://www.postgresql.org/docs/current/rangetypes.html
```
tsrange — Range of timestamp without time zone
tstzrange — Range of timestamp with time zone
```

why do we need a time zone anyway? isn't everything UTC?

Reply:
i think we can use tstzrange

Reply:
the only reason why i wanted to use timezone was b/c the other tables that are in clutch (auditing) use it. Eventually when we have scheduling it may matter but right now it probably doesn't.

Reply:
scheduling (using start_time and end_time) will be a part of the initial implementation of `mobile_experimentation` package so lets make sure that we are not blocking ourselves. That being said, both timestamp and timestamp with time zone should work here - from the perspective of the end user a frontend will always keep using a local time zone and we can convert to UTC as needed (if we decided against timestamp with timezone).

Reply:
ok, i switched to tstzrange